vcell-rest/src/main/java/org/vcell/restq/db/UserRestDB.java

```diff
@@ -42,22 +43,34 @@ public UserRestDB(AgroalConnectionFactory agroalConnectionFactory) throws DataAc
 
 
     // TODO: add some short-lived caching here to avoid repeated database calls
-    public User getUserFromIdentity(SecurityIdentity securityIdentity, boolean defaultGuest) throws DataAccessException {
+
+    /**
+     * Get the users identity from the requests that they make, usually this is a JWT token, placed within the
+     * HTTP Authorization header. If defaultGuest is true, then null will be returned for anonymous users and guests, otherwise
+     * and error will be thrown about the user being unauthenticated.
+     * Tokens generated by the old API for guest users will have no effect on this function because the securityIdentity
```

Comment:
remove any notion of a guest user please ... on an endpoint by endpoint basis, one may pass down a DUMMY_USER, but this should not be a global strategy.  Really this is because we don't have roles and authorization for writing - other